css/index.css

```diff
@@ -1 +1,711 @@
+    flex-wrap: wrap;
+    padding: 20 20;
+    margin-left: 10px;
+    margin-right: 10px;
+    margin-bottom: 30px;
+}
+
+.container3 .footer-product {
+    display: flex;
+    margin-top: 30px;
+    margin-bottom: 100px;
+}
+
+.container3 .footer-product .i7 {
+    width: 20%;
+    margin-left: 550px;
+}
+
+.container3 .copyright {
+    text-align: center;
+    margin-bottom: 40px;
+}
+
+@media (max-width: 800px) {
+    html {
+        font-size: 62.5%
+    }
+
+    .container {
+        max-width: 880px;
+        margin: 0 auto;
+        width: 90%;
+    }
+
+    .container header {
+        display: flex;
+        flex-direction: column-reverse;
+        height: 8rem;
+        width: 100%;
+    }
+
+    .container header .logo {
+        align-self: center;
+        margin-top: .3rem;
+    }
+
+    .container header nav {
+        font-size: 1.25rem;
+        display: flex;
+        width: 100%;
+    }
+
+    .container header nav a {
+        padding-bottom: .2em;
+        margin-left: 0px;
+        width: 100%;
+    }
+
+    .container .cta .main-img {
+        display: none;
+    }
+
+    .container .cta {
+        height: 29.5rem;
+        margin-bottom: 0px;
+        margin-left: 0xp;
+        padding-bottom: 0px;
+        width: 100%;
+    }
+
+    .container .cta .cta-text {
+        display: flex;
+        align-items: center;
+        margin-top: 1.7rem;
+        margin-bottom: 15rem;
+        font-size: 59px;
+        height: 27rem;
+        width: 100%;
+        margin-left: 0px;
+    }
+
+    .container .cta .cta-text button {
+        font-size: 1.3rem;
+        width: 13.5rem;
+        margin-top: 1.7rem;
+        height: 2.6rem;
+        margin-left: 0px;
+    }
+
+    .container .main-content {
+        border-top: 1px solid black;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+        margin-top: 0rem;
+        font-size: 1.3rem;
+        padding-top: 2.8rem;
+    }
+
+    .container .main-content .middle-img {
+        width: 100%;
+        margin-top: 3.6rem;
+        margin-bottom: 0rem;
+    }
+
+    .container .main-content h4 {
+        padding-bottom: 1.5rem;
+        font-size: 1.3rem;
+    }
+
+    .container .main-content .bottom-content {
+        margin-top: 1.9rem;
+    }
+
+    .container .contact {
+        width: 100%;
+        margin-bottom: 0rem;
+    }
+}
+
+@media (max-width: 500px) {
+    html {
+        font-size: 62.5%
+    }
+
+    .container {
+        max-width: 880px;
+        margin: 0 auto;
+        width: 100%;
+    }
+    .container header {
+        width: 100%;
+        display: flex;
+        flex-direction: column-reverse;
+        height: 20rem;
+        margin-bottom: 8.2rem;
+    }
+    .container header .logo {
+        padding-bottom: 2rem;
+    }
+    .container header nav {
+        flex-direction: column;
+        text-align: center;
+        height: 20rem;
+    }
+    .container header nav a {
+        padding: 1rem 0rem;
```

Comment:
This padding is what's pushing the a tags outside of it's parent container. How can we get the desired effect without thie negative side-effect?